framework/src/components/logger/logger.js

```diff
@@ -58,7 +65,7 @@ const createLogger = ({ fileLogLevel, consoleLogLevel, logFileName }) => {
 			: [];
 	const streams = [...consoleStream, ...fileStream];
 	return bunyan.createLogger({
-		name: 'lisk-framework',
+		name,
```

Comment:
I suggest to have a bound field `module` or `tag` to identify user specific identifier.

Reply:
Add an extra property `module` (in class module) with shorthand module alias to used as `module` name.